libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/NullEx.kt

```diff
@@ -1,3 +1,5 @@
+@file:JvmName("NullEx")
```

Comment:
These are not extension functions. What about `NullUtils` instead? 👀 https://github.com/mapbox/mapbox-navigation-android/pull/2098#discussion_r331560260 😅